drivers/mtd/ftl.c

```diff
@@ -120,6 +126,56 @@ static const struct block_operations g_bops =
  * Private Functions
  ****************************************************************************/
 
+static int ftl_init_map(FAR struct ftl_struct_s *dev)
```

Comment:
Please include the Description comment above this ftl_init_map() function

Reply:
Done~